FabNESO/tasks.py

```diff
@@ -37,10 +106,20 @@ def neso(
         solver: Which NESO solver to use.
         conditions_file_name: Name of conditions XML file in configuration directory.
         mesh_file_name: Name of mesh XML in configuration directory.
+        processes: Number of processes to run on.
```

Comment:
Sounds like this was written for 'cores' and then replaced with processes - would `Number of processes to run.` sound more accurate?

Reply:
Agreed your suggestion is better, will update!